### PR TITLE
Put core outside of src + make *.ctrl services

### PR DESCRIPTION
**Pull request by klumba12**

### CODE REVIEW

File: src/view/components/body/td.core.js [hunk 1]
```diff
@@ -2,8 +2,9 @@ import Directive from '@grid/view/directives/directive';
 import cellBuilder from '../cell/cell.build';
 import {AppError} from '@grid/core/infrastructure';
 import {VIEW_CORE_NAME, TD_CORE_NAME, GRID_NAME} from '@grid/view/definition';
-import {GRID_PREFIX} from '@grid/core/definition';
-import * as css from '@grid/core/services/css';
+import {TdCtrl} from '@grid/core/cell/td.ctrl';
+ 
+const classify = TdCtrl.classify;
```
Comment (vkorolev) [inline src/view/components/body/td.core.js:7]: Somewhere `TdCtrl` methods are called through class reference e.g `TdCtrl.classify(...)`, and somewhere through the variable: `const classify = TdCtrl.classify` .
I think it should be unified

File: core/style/style.view.d.ts [hunk 2]
```diff
@@ -20,5 +20,6 @@ export declare class StyleView extends View {
 	valueFactory: IValueFactory;
 	active: IActive;
 	monitor: IMonitor;
-	invalidate(): void;
+	invalidate(any, any): void;
```
Comment (titovmx) [inline core/style/style.view.d.ts]: argument names?

File: bower.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "ng-qgrid",
-	"version": "1.0.63",
+	"version": "1.3.0",
```
Comment (titovmx) [inline bower.json]: Are you going to release new version?

File: src/view/components/body/td.core.js (hunk 1 above)
Comment (klumba12) [reply src/view/components/body/td.core.js:7]: yes but here i need performance we can think about this in the future

**Review by vkorolev — APPROVED** [review]

**Review by titovmx — APPROVED** [review]

**Pull request merged**